doc/api/packages.md

```diff
@@ -607,22 +608,28 @@ Runtimes or tools other than Node.js can use them at their discretion.
 
 These user conditions can be enabled in Node.js via the [`--conditions` flag][].
 
-The following condition definitions are currently endorsed by Node.js:
+The following condition definitions are currently endorsed by Node.js, and
+are _listed in order from most to least specific, as conditions should be
+used._
 
-* `"browser"` - any environment which implements a standard subset of global
-  browser APIs available from JavaScript in web browsers, including the DOM
-  APIs.
+* `"types"` - can be used by typing systems to resolve the typing file for
+  the given export, possible since the interface should be the same for all
+  variations. _This condition should always be included first._
+* `"deno"` - indicates a variation for the Deno platform.
+* `"browser"` - any web browser environment.
 * `"development"` - can be used to define a development-only environment
-  entry point. _Must always be mutually exclusive with `"production"`._
+  entry point, for example to provide additional debugging context such as
+  better error message when running in a development mode. _Must always be
+  mutually exclusive with `"production"`._
 * `"production"` - can be used to define a production environment entry
   point. _Must always be mutually exclusive with `"development"`._
 
 The above user conditions can be enabled in Node.js via the
 [`--conditions` flag][].
 
-Platform specific conditions such as `"deno"`, `"electron"`, or `"react-native"`
-may be used, but while there remain no implementation or integration intent
-from these platforms, the above are not explicitly endorsed by Node.js.
+Platform specific conditions such as `"electron"`, or `"react-native"` may
+be used, but while there remain no implementation or integration intent
+from these platforms, we do not yet explicitly recommend them.
```

Comment:
```suggestion
from these platforms, they are not explicitly endorsed by Node.js.
```

Reply:
This was as suggested by @Trott in https://github.com/nodejs/node/pull/40708#discussion_r741622618, I'm open to a suggestion that takes into account the original feedback.

Reply:
If you wanted to make the case that "endorse" is sufficiently different from "recommend" that my original comment doesn't apply, I wouldn't argue the point.

Reply:
I just intended to restore the passive voice from the original version. I don't care re recommend vs endorse.

Reply:
Ok I've landed the passive voice with recommended!